tests/test_helper/test_helper.py

```diff
@@ -0,0 +1,141 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+import argparse
+import boto3
+import sys
+import os
+
+
+s3 = boto3.resource('s3')
+s3_client = boto3.client('s3')
+
+
+s3_control_client = boto3.client('s3control')
+
+
+BUCKET_NAME = 'aws-c-s3-test-bucket'
+# Create a public bucket with one object for testing public access
+PUBLIC_BUCKET_NAME = 'aws-c-s3-test-bucket-public'
+
+REGION = 'us-west-2'
+os.environ['AWS_DEFAULT_REGION'] = REGION
```

Comment:
`us-west-2` is hard-coded in the test, we can try to avoid that when we add the support to set all those path, bucket and stuff for tests from enc-variable or something.